packages/strapi-parts/src/Searchbar/Searchbar.stories.mdx

```diff
@@ -13,27 +13,29 @@ import { SearchForm } from './SearchForm';
 
 # Searchbar
 
-SearchBars are used to search and navigate through specific content. 
+SearchBars are used to search and navigate through specific content.
 
 **Best practices**
-
 - SearchBar should not be used at a global application level.
 - The search icon or the cancel one should not be changed.
 - SearchBar should always have a "Search" placeholder.
 - SearchBar should be placed above the content to search into.
-- SearchBar do not have a visual label but hidden details should be provided for accessibility matters. 
+- SearchBar do not have a visual label but hidden details should be provided for accessibility matters.
 - If adding a SearchBar, make sure to add a "No results found" state.
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/Searchbar)
+
 
 ## Imports
 
 ```js
-import { Searchbar, SearchForm } from '@strapi/parts/Searchbar';
+import { Searchbar } from '@strapi/parts/Searchbar';
```

Comment:
`import { Searchbar, SearchForm } from '@strapi/parts/Searchbar';`

Reply:
I dont see where the searchform is being exported in the searchbar.js so that's why i added a second export. could you explain where is comes from?